### ISSUE TITLE
Browse without Signing Up

### ISSUE THREAD

**Issue by JasonHighVibe** [HighVibe/hvn#31]
Outcome:
User will be able to go directly into the App discover and see which streams are live; even viewing them without having to sign-up.

Users will also be able to access the stream inbrowser given a direct link.

However, if the user wants to comment, begin earning, or create any content, they must sign-up. 

Purpose:
Provide value for everyone even when not directly signed up for the App. Will function better than Zoom (because you wont have to download)

**Comment by 7flash** [comment]
Yes!

We aim to make everything transparent and easy to access (blockchain public access)

Then we allow users to create blockchain account to have own profile, earn coins, and make contributions (blockchain write access)

**Comment by 7flash** [comment]
In this approach, then there is no authorization with email/social networks/phone number

Only anonymous public access or authorized personal access

User have opportunity to browse application without creation of account, but he will need to create account in order to join tribe, participate in discussion, and have access to all other features.

User is able to create blockchain account for free, by verifying his phone number with sms code, and then his private key will be saved in local preferences of device.

Telos account of user is his identity, and his private key is his passport. All interactions are being made from his account name, and signature should be provided as proof of key ownership.

In first version, application may interact firebase directly, but in later release, more things should go through smart contracts, and then data from blockchain actions being converted to firebase by reducer service.

**Comment by JasonHighVibe** [comment]
Yes exactly! Important to have this process work seamlessly without overburdening user during sign-up process. I like the flow you outlined.

And let's keep things through firebase initially - i like that approach.
